<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Excel 查找工具</string>
  </property>

  <!-- 待搜索文件 -->
  <widget class="QWidget" name="widget_search_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_search_file">
    <item>
     <widget class="QLabel" name="label_search_file_tag">
      <property name="text">
       <string>待搜索文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_search_file"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_browse_search">
      <property name="text">
       <string>浏览...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>

  <!-- 数据源文件 -->
  <widget class="QWidget" name="widget_target_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_target_file">
    <item>
     <widget class="QLabel" name="label_source_file_tag">
      <property name="text">
       <string>数据源文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_target_file"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_browse_target">
      <property name="text">
       <string>浏览...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>

  <!-- 导出文件路径 -->
  <widget class="QWidget" name="widget_output_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_output_file">
    <item>
     <widget class="QLabel" name="label_export_file_tag">
      <property name="text">
       <string>导出文件夹</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edit_output_file"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_browse_output">
      <property name="text">
       <string>浏览...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>

  <!-- 开始搜索按钮 -->
  <widget class="QPushButton" name="btn_run">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>262</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始搜索</string>
   </property>
  </widget>

  <!-- 输出文本框 -->
  <widget class="QTextEdit" name="txt_output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>521</width>
     <height>80</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
